site/src/sphinx/server-annotated-service.rst

```diff
@@ -0,0 +1,627 @@
+.. _@Put: apidocs/index.html?com/linecorp/armeria/server/annotation/Put.html
+.. _@RequestConverter: apidocs/index.html?com/linecorp/armeria/server/annotation/RequestConverter.html
+.. _@RequestObject: apidocs/index.html?com/linecorp/armeria/server/annotation/RequestObject.html
+.. _@ResponseConverter: apidocs/index.html?com/linecorp/armeria/server/annotation/ResponseConverter.html
+.. _@Trace: apidocs/index.html?com/linecorp/armeria/server/annotation/Trace.html
+.. _AggregatedHttpMessage: apidocs/index.html?com/linecorp/armeria/common/AggregatedHttpMessage.html
+.. _ByteArrayRequestConverterFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/ByteArrayRequestConverterFunction.html
+.. _DecoratingServiceFunction: apidocs/index.html?com/linecorp/armeria/server/DecoratingServiceFunction.html
+.. _ExceptionHandlerFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/ExceptionHandlerFunction.html
+.. _HttpResponse: apidocs/index.html?com/linecorp/armeria/common/HttpResponse.html
+.. _HttpResponseException: apidocs/index.html?com/linecorp/armeria/server/HttpResponseException.html
+.. _HttpStatusException: apidocs/index.html?com/linecorp/armeria/server/HttpStatusException.html
+.. _JacksonRequestConverterFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/JacksonRequestConverterFunction.html
+.. _PathMapping: apidocs/index.html?com/linecorp/armeria/server/PathMapping.html
+.. _RequestConverterFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/RequestConverterFunction.html
+.. _ResponseConverterFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/ResponseConverterFunction.html
+.. _ServerBuilder: apidocs/index.html?com/linecorp/armeria/server/ServerBuilder.html
+.. _Service: apidocs/index.html?com/linecorp/armeria/server/Service.html
+.. _StringRequestConverterFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/StringRequestConverterFunction.html
+
+.. _server-annotated-service:
+
+Annotated HTTP Service
+======================
+
+*Armeria* provides a way to write an HTTP service using annotations. It helps a user to make his or her codes
+simple and easy to understand. A user is able to run an HTTP service by couple of lines codes using
+annotations as follows. ``getHello()`` method in the example would be mapped to the path of ``/hello``
+with an HTTP GET method.
+
+.. code-block:: java
+
+    ServerBuilder sb = new ServerBuilder();
+    sb.annotatedService(new Object() {
+        @Get("/hello/{name}")
+        public HttpResponse hello(@Param("name") String name) {
+            return HttpResponse.of(HttpStatus.OK,
+                                   MediaType.PLAIN_TEXT_UTF_8,
+                                   "Hello, %s!", name);
+        }
+    });
+
+Mapping HTTP service methods
+----------------------------
+
+To map each method in an annotated HTTP service class to HTTP path, both `@Path`_ annotation and HTTP method
```

Comment:
- each method -> a Java method
- both .. annotation ... are used -> it has to be annotated with both ...
- Could we start from the simpler case where no `@Path` annotation is required instead? We could even explain our path mapping syntax without introducing `@Path` annotation at all and then explain how to handle multiple HTTP method.